test/back_end/api_get_test.spec.js

```diff
@@ -210,8 +264,12 @@ describe("Test GET requests to API", function() {
     });
     it("should return list of tas under Elizabeth Miller", async function() {
         try {

```

Comment:
Again...it is silly to have these in `try` `catch` blocks...

Reply:
Sorry. It was not written by me. But I will revise that.